### ISSUE TITLE
TypeError: t.target is undefined.

### ISSUE THREAD

**Issue by DutchEclipse** [ActivityWatch/activitywatch#721]
- [x] I am on the [latest](https://github.com/ActivityWatch/activitywatch/releases/latest) ActivityWatch version.
 - [x] I have searched the issues of this repo and believe that this is not a duplicate.

- **OS name and version**: Windows 11 21H1 OS Build 22000 493
- **ActivityWatch version**: v0.11.0
- **Firefox Version 98.0

## Describe the bug
"TypeError: t.target is undefined. See dev console (F12) and/or server logs for more info." error when changing settings of Activitywatch

## Expected behavior
The setting should save without issue.

## Documentation
<!--
Screenshot of the error
![image](https://user-images.githubusercontent.com/83969413/157940336-57b37c32-17d4-4785-a85d-3d34c9910a1f.png)

[Log files](https://we.tl/t-YqeSKLHrpx)
Usable for a week.

**Comment by ErikBjare** [comment]
This has been reported numerous times before. Try the v0.12.0b1 beta release, which has the issue fixed.

**Comment by DutchEclipse** [comment]
> This has been reported numerous times before. Try the v0.12.0b1 beta release, which has the issue fixed.

Bug is still there in version v0.12.0b1

![image](https://user-images.githubusercontent.com/83969413/158910931-94c27ecb-3e4e-4c3d-acd8-71ac8f1871dc.png)
![image](https://user-images.githubusercontent.com/83969413/158910958-41e2c989-f58a-49fe-a066-2a73e772b6c0.png)

**Comment by ErikBjare** [comment]
No, you just have a stale browser cache. Try force-refreshing the page.
